### PR TITLE
Lt 920 finalised cases

### PR DESCRIPTION
**Pull request by stephanieboyd**

### CODE REVIEW

File: templates/case/hmrc-case.html [hunk 1]
```diff
@@ -36,7 +36,7 @@ <h2 class="govuk-heading-l">{{ title }}</h2>
 			<a role="button" draggable="false" class="govuk-button" href="{% url 'cases:manage' case.id %}">
 				Change Status
 			</a>
-			{% if permissions|key_in_array:'MAKE_FINAL_DECISIONS' %}
+			{% if permissions|key_in_array:'MAKE_FINAL_DECISIONS' and status not in terminal_statuses%}
```
Comment (Mark-Spiers-kainos) [inline templates/case/hmrc-case.html]: terminal_case_statuses?

File: templates/case/team-advice-view.html [hunk 2]
```diff
@@ -36,9 +36,11 @@ <h2 class="govuk-heading-l">{{ title }}</h2>
                         </button>
                     </div>
                 {% else %}
-                    <a class="govuk-button" href="{% url 'cases:coalesce_user_advice' case.id %}" class="lite-tabs__tab" id="combine_advice_button">
-                        Combine all user advice
-                    </a>
+                    {% if case.application.status.key not in terminal_case_statuses%}
+                        <a class="govuk-button" href="{% url 'cases:coalesce_user_advice' case.id %}" class="lite-tabs__tab" id="combine_advice_button">
+                            Combine all user advice
```
Comment (andytaylorkainos) [inline templates/case/team-advice-view.html]: content string :)

File: templates/case/user-advice-view.html [hunk 3]
```diff
@@ -27,15 +27,17 @@ <h2 class="govuk-heading-l">{{ title }}</h2>
 			</div>
             {% if not case.has_advice.my_team %}
                 {% if not case.has_advice.final %}
-                    <button id="button-give-advice" type="submit" class="govuk-button" data-module="govuk-button">
-                        Give or change advice
-                    </button>
+                    {% if case.application.status.key not in terminal_case_statuses%}
+                        <button id="button-give-advice" type="submit" class="govuk-button" data-module="govuk-button">
+                            Give or change advice
```
Comment (andytaylorkainos) [inline templates/case/user-advice-view.html]: Content string :)

File: cases/views/main.py [hunk 4]
```diff
@@ -168,14 +175,30 @@ def get(self, request, **kwargs):
 
         reduced_statuses = {
             "statuses": [
-                x for x in statuses["statuses"] if (x["status"] != "finalised" and x["status"] != "applicant_editing")
+                status
+                for status in statuses["statuses"]
+                if status["status"] == "closed" or status["status"] == "submitted" or status["status"] == "withdrawn"
```
Comment (Mark-Spiers-kainos) [inline cases/views/main.py]: could you use another in here?

File: cases/views/main.py [hunk 5]
```diff
@@ -168,14 +175,30 @@ def get(self, request, **kwargs):
...
+            # additional but still reduced statuses needed for applications
+            reduced_statuses = {
+                "statuses": [
+                    status
+                    for status in statuses["statuses"]
+                    if (
+                        status["status"] != "finalised"
+                        and status["status"] != "applicant_editing"
+                        and status["status"] != "closed"
+                        and status["status"] != "registered"
```
Comment (Mark-Spiers-kainos) [inline cases/views/main.py]: same as above, but "not in"

**Pull request merged**